packages/experimental/Badge/src/Badge.styling.ts

```diff
@@ -13,29 +15,38 @@ import { UseStylingOptions, buildProps, Theme } from '@fluentui-react-native/fra
 import { borderStyles, layoutStyles, fontStyles } from '@fluentui-react-native/tokens';
 import { defaultBadgeTokens } from './BadgeTokens';
 import { defaultBadgeColorTokens } from './BadgeColorTokens';
+import { badgeFontTokens } from './BadgeFontTokens';
 
 export const coreBadgeStates: (keyof BadgeCoreTokens)[] = [...BadgeSizes, ...BadgeShapes];
 export const badgeStates: (keyof BadgeTokens)[] = [...coreBadgeStates, ...BadgeColors, ...BadgeAppearances];
+const tokensThatAreAlsoProps: (keyof BadgeConfigurableProps)[] = ['badgeColor', 'position'];
 
 export const stylingSettings: UseStylingOptions<BadgeProps, BadgeSlotProps, BadgeTokens> = {
-  tokens: [defaultBadgeTokens, defaultBadgeColorTokens, badgeName],
+  tokens: [defaultBadgeTokens, defaultBadgeColorTokens, badgeFontTokens, badgeName],
   states: badgeStates,
+  tokensThatAreAlsoProps,
   slotProps: {
     root: buildProps(
-      (tokens: BadgeTokens, theme: Theme) => ({
-        style: {
-          ...getBadgePosition(tokens),
-          alignItems: 'center',
-          flexDirection: 'row',
-          alignSelf: 'flex-start',
-          justifyContent: 'center',
-          minHeight: tokens.minHeight,
-          width: tokens.width,
-          backgroundColor: tokens.backgroundColor,
-          ...borderStyles.from(tokens, theme),
-          ...layoutStyles.from(tokens, theme),
-        },
-      }),
+      (tokens: BadgeTokens, theme: Theme) => {
+        const { badgeColor, backgroundColor, position } = tokens;
+        const _badgeColor = !badgeColor || BadgeColors.includes(badgeColor as BadgeNamedColor) ? backgroundColor : badgeColor;
+        const _position = position ?? tokens.position;
```

Comment:
> const _position = position ?? tokens.position; [](http://example.com/codeflow?start=7&length=47)

aren't position and tokens.position the same?

Reply:
oh, thanks for catching this :)